examples/vulkan/Vulkan.cpp

```diff
@@ -1455,13 +1456,13 @@ class VulkanExample
         }
 
         // Copy the vertex data into the buffer
-        std::memcpy(ptr, vertexData.data(), sizeof(vertexData));
+        std::memcpy(ptr, vertexData.data(), vertexData.size());
 
         // Unmap the buffer
         vkUnmapMemory(device, stagingBufferMemory);
 
         // Create the GPU local vertex buffer
-        if (!createBuffer(sizeof(vertexData),
+        if (!createBuffer(vertexData.size(),
```

Comment:
This is not correct, this gives the vertex count but not the size of the vertex this should be multiplied by sizeof(vertexData[0]); or continue using sizeof